utils/file.go

```diff
@@ -21,6 +21,9 @@ func ReadOffset(r io.Reader) (uint64, error) {
     log.Debug().Msg("reading offset")
     var offset uint64
     _, err := fmt.Fscan(r, &offset)
+    if err == io.EOF { // if file empty just set 0
```

Comment:
handler should decide whether to do it, function should return an error

Reply:
fixed